packages/portfolio-deploy/test/portfolio.test.ts

```diff
@@ -466,4 +455,283 @@ test.serial('restart contract', async t => {
   );
 });
 
+test.serial('CCTP settlement works across contract restarts', async t => {
+  const { walletFactoryDriver: wfd } = t.context;
+
+  const myMarshaller = makeClientMarshaller(v => (v as any).getBoardId());
+  const wallet = await wfd.provideSmartWallet(beneficiary, myMarshaller);
+
+  await wallet.executeOffer({
+    id: '456',
+    invitationSpec: {
+      source: 'continuing',
+      previousOffer: 'settle-cctp',
+      invitationMakerName: 'SettleCCTPTransaction',
+    },
+    proposal: { give: {}, want: {} },
+    offerArgs: {
+      txDetails: {
+        amount: 40_000n,
+        remoteAddress: '0x126cf3AC9ea12794Ff50f56727C7C66E26D9C092',
+        status: 'confirmed',
+      },
+      remoteAxelarChain: 'eip155:42161',
+    },
+  });
+
+  const finalUpdate = wallet.getLatestUpdateRecord();
+  t.log('Final wallet update:', finalUpdate);
+
+  t.like(finalUpdate, {
+    status: {
+      id: '456',
+      invitationSpec: {
+        invitationMakerName: 'SettleCCTPTransaction',
+        source: 'continuing',
+        previousOffer: 'settle-cctp',
+      },
+      numWantsSatisfied: 1,
+      offerArgs: {
+        remoteAxelarChain: 'eip155:42161',
+        txDetails: {
+          amount: 40000n,
+          remoteAddress: '0x126cf3AC9ea12794Ff50f56727C7C66E26D9C092',
+          status: 'confirmed',
+        },
+      },
+    },
+  });
+
+  t.log('Test completed: CCTP settlement works across contract restarts');
```

Comment:
Does the `t.like(finalUpdate, …)` assertion actually cover "CCTP settlement works across contract restarts"? I think I would hope to see expected effects, rather than just a wallet update.

Reply:
Yes; here's hoping...

I think the effects are not visible until...

 - #11714

but when that's added to this PR, it would makes sense to update this test.